checkers/deprecatedComment_checker.go

```diff
@@ -114,7 +116,7 @@ func (c *deprecatedCommentChecker) VisitDocComment(doc *ast.CommentGroup) {
 
 		// Check for other commonly used patterns.
 		for _, pat := range c.commonPatterns {
-			if pat.MatchString(l) {
+			if strings.HasPrefix(strings.ToLower(l), pat) {
```

Comment:
It's possible to use EqualFold here, if you take the prefix of `l` of `len(pat)`.
This will avoid allocations in here.